agreementbot/pattern_manager.go

```diff
@@ -192,7 +192,32 @@ func (pm *PatternManager) UpdatePatternPolicies(org string, definedPatterns map[
 
 	// Exit early on error
 	if !pm.hasOrg(org) {
-		return errors.New(fmt.Sprintf("org %v not found in pattern manager", org))
+		return nil
```

Comment:
why is this not an error any more?

Reply:
I added the error back because the UpdatePatternPolicies is only called when there the org is in the pm.